yellowbrick/features/pca.py

```diff
@@ -172,26 +181,57 @@ def transform(self, X, y=None, **kwargs):
 
     def draw(self, **kwargs):
         X = self.pca_features_
-
         if self.proj_dim == 2:
             self.ax.scatter(X[:, 0], X[:, 1], c=self.color, cmap=self.colormap)
+            if self.proj_features:
+                x_vector = self.pca_components_[0]
+                y_vector = self.pca_components_[1]
+                max_x = max(X[:, 0])
+                max_y = max(X[:, 1])
+                for i in range(self.pca_components_.shape[1]):
+                    plt.arrow(
```

Comment:
We prefer the `self.ax.arrow` and `self.ax.text` methods to ensure that the correct axes is drawn upon (just for future reference, we hope you'll contribute more!). I can update this though.

Reply:
OK, sure I will have it in mind

Reply:
Thanks!